custom_subsets/elastic_endpoint/alerts/malware_event.yaml

```
@@ -469,6 +469,8 @@ fields:
                 exceptionable: true
               status:
                 exceptionable: true
+          malware_signature:
+            fields: "*"
```

Everything else here looks like this:
```suggestion
            fields: {}
```

---

We should update the docs on `"*"` vs `{}` because I always forget haha. `{}` only works when the field is a leaf aka if we only wanted `malware_signature.all_names` in the mapping we could write it like:

```
malware_signature:
  fields:
    all_names: {}
```
Since we want everything under `malware_signature` and `fields` is an array we have to use the `"*"` syntax like Chan has here.